superset/utils/log.py

```diff
@@ -224,6 +224,9 @@ def log(  # pylint: disable=too-many-arguments,too-many-locals
         logs = list()
         for record in records:
             json_string: Optional[str]
+            record.update(
+                {"path": path, "path_no_int": path_no_int, "ref": ref,}
+            )
```

Comment:
This should probably happen in `log_context` so you don't have to pass these arguments around.

Reply:
done